livestyled/schemas/product.py

```diff
@@ -36,8 +27,7 @@ class Meta:
 
     id = fields.Int()
     price = fields.Integer()
-    stocks = RelatedResourceField(schema='livestyled.schemas.product.ProductVariantStockSchema', many=True,
-                                  microservice_aware=True, missing=None)
+    stocks = fields.List(fields.String(), missing=[])
```

Comment:
No. Please return the code. A lambda will expect a ProductVariantStock json returned and not an IRI. This is what is meant to happen.

Reply:
Our RelatedResourceField doesn't expect to work with structure that I described above. Should I add support for iris with such compose_ids and extract just this part: `fulfilmentPoint=1;productVariant=5`?

I just don't get why should we remove iri, since places where we use python-sdk will receive this value to get/update product_variant_stock and we'll have to convert this value to IRI once again to make it. So our sdk removes `sell/product_variant_stocks/`  part and wherever we use it we'll have to add it once again, does it make sense?

Reply:
Why isn't this a `RelatedResourceLinkField` instead?